src/schema-gen/InspectorXSLT/testing/generate-oscal-catalog-inspector.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Simple invocation of the pipeline OSCAL-INSPECTOR-XSLT.xpl
+# which hard-wires the production of InspectorXSLT for OSCAL formats (currently catalog)
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+# shellcheck source=../common/subcommand_common.bash
```

Comment:
Minor, non-blocking: I am a big believer in removing cruft and reviewing history, but I know it is not for everyone.

```suggestion
```